docs/vgclog_examples.md

```diff
@@ -706,3 +707,895 @@ Analyzing the structure and elements of this example log output shows that this
+
+<concurrent-end/>                          (global mark cycle concurrent subincrement finishes )
+
+<allocation-taxation/>                     (memory threshold trigger recorded)
+
+<cycle-continue/>                          (change of cycle type from global mark to global)
+
+</gc-start type="global garbage collect"/> (global cycle STW increment starts)
+
+<mem-info>                                 (memory status before operations)
+
+ <mem></mem>                               (status of different types of memory)
+
+</mem-info>         
+
+</gc-start type="global garbage collect"/>
+
+<allocation-stats/>                        (Snapshot of how memory was divided up between
+                                           ... application threads before current cycle started)
+<gc-op> type="mark" </gc-op>                (mark operation completed)
+
+<gc-op> type="class unload" </gc-op>        (class unload operation completed)
+
+<gc-op> type="sweep" </gc-op>               (sweep operation completed)
+
+<gc-op> type="compact" </gc-op>             (compact operation completed)
+
+<gc-end type="global garbage collect">      (global cycle STW increment ends)
+
+<mem-info>                                  (memory status after operations)
+          
+<mem></mem>                                 (status of different types of memory)
+
+</mem-info>         
+
+</gc-end type="global garbage collect">
+
+<cycle-end type = "global garbage collect"/> (cycle ends)
+
+<allocation-satisfed/>                      (required allocation has been achieved)
+
+<exclusive-end>                             (STW pause ends)
+```
+
+The following example shows a `balanced` global cycle that is triggered during a [global mark cycle](vgclog_examples.md#balanced-global-mark-gc-cycle). 
+
+The start of the GMP work processing subincrement of the global mark cycle, which runs concurrently with application threads, is recorded using the `<concurrent-start>` element.
+
+```xml
+<concurrent-start id="2009" type="GMP work packet processing" contextid="2003" timestamp="2021-03-05T12:16:43.109">
+  <concurrent-mark-start scanTarget="18446744073709551615" />
+</concurrent-start>
+```
+
+After the start of the concurrent subincrement, The logs record an allocation failure using `<af-start>`. The `<concurrent-end>` element attribute `terminationReason` shows that a termination of the concurrent increment was requested by the garbage collector. 
+
+```xml
+<af-start id="2010" threadId="00000000008AA780" totalBytesRequested="24" timestamp="2021-03-05T12:16:43.109" intervalms="1212.727" />
+<concurrent-end id="2011" type="GMP work packet processing" contextid="2003" timestamp="2021-03-05T12:16:43.110" terminationReason="Termination requested">
+<gc-op id="2012" type="mark increment" timems="0.893" contextid="2003" timestamp="2021-03-05T12:16:43.110">
+  <trace-info scanbytes="584612" />
+</gc-op>
+</concurrent-end>
+```
+
```

Comment:
I think it's unnecessary to talk about concurrent-start/end details. It detracts from important things af-start and cycle-continue/cycle-start

Reply:
I need to understand this more - given that we are stepping the user through the logs, is it that we are talking about `<concurrent-end>` info that we do not need to include? Or that you do not think I should start the global mark example by showing how the global mark increment changes type?

_After the start of the concurrent subincrement, The logs record an allocation failure using `<af-start>`. The `<concurrent-end>` element attribute `terminationReason` shows that a termination of the concurrent increment was requested by the garbage collector._

Reply:
It's definitely ok as-is. It's just felt a bit too long and too detailed to go into concurrent-end and termination reason.